mvrx-rxjava2/src/test/kotlin/com/airbnb/mvrx/ViewModelSubscriberTest.kt

```diff
@@ -781,13 +781,6 @@ class ViewModelSubscriberTest : BaseTest() {
         assertTrue(job.isCompleted)
     }
 
-    @Test
-    fun testCancelledIfViewModelCleared() = runBlocking {
```

Comment:
@elihart @denis-bezrukov I had to remove this test. I'm not sure why it still passed after the change to not cancel the store's state flow but it seems like this isn't the expected behavior anymore right?

Reply:
@gpeal  It passed because this test under the hood used another store  (sync state store via mockable state store. And sync store used to close the channel in invokeOnCancel).

Yep, it is not expected anymore